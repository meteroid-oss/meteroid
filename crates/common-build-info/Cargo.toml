[package]
name = "common-build-info"
version = "0.1.0"
rust-version.workspace = true
edition.workspace = true
license.workspace = true
# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
build-info.workspace = true

[build-dependencies]
build-info-build.workspace = true
