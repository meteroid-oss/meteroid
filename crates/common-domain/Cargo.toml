[package]
name = "common-domain"
version = "0.1.0"
rust-version.workspace = true
edition.workspace = true
license.workspace = true

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
serde = { workspace = true, features = ["derive"] }
uuid = { workspace = true, features = ["serde", "v7"] }
base62 = { workspace = true }
diesel = { workspace = true, features = ["uuid", "postgres_backend"], optional = true }
diesel-derive-newtype = { workspace = true, optional = true }
utoipa = { workspace = true, optional = true }

[dev-dependencies]
serde_json = { workspace = true }

[features]
diesel = ["dep:diesel", "dep:diesel-derive-newtype"]
utoipa = ["dep:utoipa"]
default = []

