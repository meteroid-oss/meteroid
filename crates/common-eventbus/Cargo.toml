[package]
name = "common-eventbus"
version = "0.1.0"
rust-version.workspace = true
edition.workspace = true
license.workspace = true
# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
async-trait.workspace = true
thiserror.workspace = true
uuid = { workspace = true, features = ["v4", "v7"] }
chrono = { workspace = true, features = ["clock"] }

