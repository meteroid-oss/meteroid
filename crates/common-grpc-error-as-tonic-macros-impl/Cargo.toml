[package]
name = "common-grpc-error-as-tonic-macros-impl"
version = "0.1.0"
rust-version.workspace = true
edition.workspace = true
license.workspace = true

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[lib]
proc-macro = true


[dependencies]
common-grpc-error-as-tonic-macros.workspace = true
proc-macro2.workspace = true
quote.workspace = true
syn.workspace = true
