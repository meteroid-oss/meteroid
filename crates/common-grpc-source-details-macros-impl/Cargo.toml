[package]
name = "common-grpc-source-details-macros-impl"
version = "0.1.0"
rust-version.workspace = true
edition.workspace = true
license.workspace = true

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[lib]
proc-macro = true
proc-macro-derive = true


[dependencies]
common-grpc-source-details-macros.workspace = true

quote = "1.0.35"
syn = "2.0.48"
