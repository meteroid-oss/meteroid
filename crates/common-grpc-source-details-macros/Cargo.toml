[package]
name = "common-grpc-source-details-macros"
version = "0.1.0"
rust-version.workspace = true
edition.workspace = true
license.workspace = true

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html


[dependencies]
tonic.workspace = true
serde = { workspace = true, features = ["derive"] }
