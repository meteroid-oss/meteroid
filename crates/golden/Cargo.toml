[package]
name = "golden"
version = "1.0.0"
rust-version.workspace = true
edition.workspace = true
license.workspace = true

[dependencies]
serde = { workspace = true, features = ["derive"] }
serde_json.workspace = true
paste.workspace = true
